site/app/models/PollModel.php

```diff
@@ -15,8 +15,9 @@ class PollModel extends AbstractModel {
     protected $user_responses;
     protected $release_date;
     protected $status;
+    protected $image_path;
```

Comment:
You should mark this as a `/** @prop-read */` and can then delete `getImagePath()` function and take advantage of the automatically provided getters from `AbstractModel` as described [here](https://submitty.org/developer/developing_the_php_site/models).